cypress/cypress/support/functions.ts

```diff
@@ -1,6 +1,10 @@
 import { Mentor, _Ref } from "./types";
 import { login as loginDefault } from "../fixtures/login";
 import { mentorDefault } from "../fixtures/mentor";
+import { TrainStatus, VideoStatus, UserAccessToken } from "./types";
+
+const TRAIN_STATUS_URL = `/classifier/train/status`;
```

Comment:
these should come from config (config can set them, and let these literals be defaults)

Reply:
@beatthat this is in cypress. The value is a default, the cypress functions take params.statusUrl or use the constant otherwise.

Reply:
oh right! my bad